frontend/address-book/src/states.js

```diff
@@ -7,4 +7,19 @@ export const searchInputState = atom({
 export const currentSearchTerm= atom({
```

Comment:
Can you fix your friend's formatting here and add a space after `atom`?

Reply:
This has been updated in the latest push.